static/js/apps/nl_interface/app.tsx

```diff
@@ -90,19 +90,27 @@ export function App(): JSX.Element {
     setLoading(true);
     setUrlParams(urlParams);
     axios.get(`/nl/data?${urlParams}`).then((resp) => {
+      if (
+        resp.data["context"] === undefined ||
+        resp.data["config"] === undefined
+      ) {
+        setLoading(false);
+        return;
+      }
+      const context = resp.data["context"];
       setChartsData({
         place: {
-          types: [resp.data["place_type"]],
-          name: resp.data["place_name"],
-          dcid: resp.data["place_dcid"],
+          types: [context["place_type"]],
+          name: context["place_name"],
+          dcid: context["place_dcid"],
         },
         config: resp.data["config"],
       });
-      if (resp.data["debug"] === undefined) {
+      if (context["debug"] === undefined) {
```

Comment:
merge this into the if at line 93?

Reply:
Well, I was thinking that if the "debug" info is not available, that's still Ok, right? The frontend charts rendering is not dependent on "debug".

Reply:
That makes sense.